ui/app/AppLayouts/Chat/panels/communities/CommunityMembersTabPanel.qml

```diff
@@ -0,0 +1,115 @@
+
+    Layout.fillWidth: true
+    Layout.fillHeight: true
+
+    ColumnLayout {
+        anchors.fill: parent
+
+        StatusBaseInput {
+            id: memberSearch
+            Layout.preferredWidth: 351
+            rightPadding: 0
+            placeholderText: root.placeholderText
+            icon.name: "search"
+            enabled: model.count > 0
+        }
+
+        ListView {
+            id: membersList
+
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+
+            model: root.model
+            clip: true
+
+            delegate: StatusMemberListItem {
+                id: memberItem
+
+                readonly property bool itsMe: model.pubKey.toLowerCase() === userProfile.pubKey.toLowerCase()
+                readonly property bool isHovered: !memberItem.itsMe && memberItem.sensor.containsMouse
+
+                components: [
+                    StatusButton {
+                        visible: (root.panelType === CommunityMembersTabPanel.TabType.AllMembers) && isHovered
+                        text: qsTr("Kick")
+                        type: StatusBaseButton.Type.Danger
+                        size: StatusBaseButton.Size.Small
+                        onClicked: root.kickUserClicked(model.pubKey, model.displayName)
+                    },
+
+                    StatusButton {
+                        visible: (root.panelType === CommunityMembersTabPanel.TabType.AllMembers) && isHovered
+                        text: qsTr("Ban")
+                        type: StatusBaseButton.Type.Danger
+                        size: StatusBaseButton.Size.Small
+                        onClicked: root.banUserClicked(model.pubKey, model.displayName)
+                    },
+
+                    StatusButton {
+                        visible: (root.panelType === CommunityMembersTabPanel.TabType.BannedMembers) && isHovered
+                        text: qsTr("Unban")
+                        type: StatusBaseButton.Type.Normal
+                        size: StatusBaseButton.Size.Small
+                        onClicked: root.unbanUserClicked(model.pubKey)
+                    }
+                ]
+
+                width: membersList.width
+                visible: memberSearch.text === "" || title.toLowerCase().includes(memberSearch.text.toLowerCase())
+                height: visible ? implicitHeight : 0
+                color: "transparent"
+
+                pubKey: Utils.getElidedCompressedPk(model.pubKey)
+                nickName: model.localNickname
+                userName: {
+                    let tempUserName = model.displayName
+                    if (memberItem.itsMe) {
+                        tempUserName += qsTr(" (You)")
+                    }
+                    return tempUserName
+                }
+                status: model.onlineStatus
+                icon.color: Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(model.pubKey)] // FIXME: use model.colorId
```

Comment:
Can we resolve this TODO now?

Reply:
@jrainville I tried but it seems that colorId is wrong in a model. This FIXME comment is placed in a few places in the code. It is a bigger problem and I will create a separate ticket for this.